nuget/compatible-version/project/project.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include=".\library.csproj" />
```

Comment:
was the extra `.` was intentional?

Reply:
I suspect the original intention of this line was a reference to the other library project, but the relative path is incorrect.  Ultimately it didn't make a difference for this test because that file wasn't relevant to the update, but I thought it would be better to have a correct project file in case we ever need to `dotnet restore`, etc. locally.